analysis/Scripts/NaiveYieldPlot.py

```diff
@@ -1,10 +1,8 @@
 import sys, getopt, ROOT, math, textwrap
 from array import array
-import Imports
+from Imports import OUTPUT_DICT_PATH, PLOT_PATH
 
-Base_Path = "/data/bfys/jdevries/LcAnalysis_plots/"
-Import_Path = Base_Path + "Dict_output"
```

Comment:
is this not used anywhere?

Reply:
no because it can completely be replaced with PLOT_PATH and DICT_OUTPUT_PATH